lib/config/defaults.js

```diff
@@ -1435,6 +1435,14 @@ const getResolveDefaults = ({ cache, context, targetProperties, mode }) => {
 		extensions: [...jsExtensions]
 	});
 
+	/** @type {function(): ResolveOptions} */
+	const styleDeps = () => ({
+		conditionNames: ["style"],
+		mainFields: ["style", "main", "..."],
+		mainFiles: ["index", "..."],
+		extensions: [".css", "..."]
```

Comment:
Not sure about `[".css", "..."]`, because it will allow to use `@import url("file.js");`, it is wrong, so we need to add test cases, also please add testcases on each property here

Reply:
Based on it https://github.com/webpack/webpack/blob/d61b29b0901d40447dce6174b7ca608aeb18d14c/lib/config/defaults.js#L1417, there is only `css` should be